entries/controlgroup.xml

```diff
@@ -31,15 +31,15 @@
 			<xi:include href="../includes/classes-option-desc.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/classes-option-example.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 		</option>
-		<option name="direction" default='"horizontal"' example-value='{ direction: "vertical" }'>
+		<option name="direction" default='"horizontal"' example-value='"vertical"'>
 			<desc>
-				<p>By default, controlgroup displays its widget's in a horizontal layout. Use this option to use a vertical layout instead.</p>
+				<p>By default, controlgroup displays its controls in a horizontal layout. Use this option to use a vertical layout instead.</p>
 			</desc>
 			<type name="String" />
 		</option>
 		<xi:include href="../includes/widget-option-disabled.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 		<option name="onlyVisible" default="true" example-value="false">
-			<desc>Sets whether to exclude invisible children in the assignment of rounded corners. When set to false, all children of a controlgroup are taken into account when assigning rounded corners, including hidden children. Thus, if, for example, the controlgroup's first child is hidden, the controlgroup will, in effect, not have rounded corners on the top edge.</desc>
+			<desc>Sets whether to exclude invisible children in the assignment of rounded corners. When set to false, all children of a controlgroup are taken into account when assigning rounded corners, including hidden children. Thus, if, for example, the controlgroup's first child is hidden, the controlgroup will, in effect, not have rounded corners on the left edge.</desc>
```

Comment:
Makes sense for the default `direction`. Could be made explicit though, something like "not have rounded corners on the left edge (for `direction: "horizontal"`)."